CONTRIBUTORS.md

```diff
@@ -123,6 +123,8 @@ The following were contributed by Jianling Zhong. Thanks, Jianling!
 * `Add supports for a "required parameters" check`
 
 The following were contributed by Charlie Summers. Thanks, Charlie!
+* `Upgrade li-hadoop-plugin for Flow 2.0`
+* `Refactor YamlCompiler to fit closer with AzkabanDslCompiler`
```

Comment:
Thanks for updating the naming of yamlCompiler. It'll be great to update it everywhere including the comments and change description.

Reply:
Unfortunately that's a commit name so I can't change it without re-writing commit history :upside_down_face:. Sorry! But yes, this is a good point.